plugins/channelrx/demodils/ilsdemodsink.cpp

```diff
@@ -31,7 +31,6 @@
 ILSDemodSink::ILSDemodSink(ILSDemod *ilsDemod) :
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok